studio-android/LightNovelLibrary/app/src/main/java/org/mewx/wenku8/reader/activity/Wenku8ReaderActivityV1.java

```diff
@@ -470,7 +470,7 @@ public void onSingleTap(MotionEvent event) {
                     int x = (int) event.getX();
                     int y = (int) event.getY();
 
-                    if(x > screenWidth / 3 && x < screenWidth * 2 / 3 && y > screenHeight / 3 && y < screenHeight * 2 / 3) {
+                    if(x > screenWidth / 3 && x < screenWidth * 2 / 3) {
```

Comment:
Please revert this change, it changes the behaviour for page turning.

Previously, the screen was divided into this shape:

![image](https://user-images.githubusercontent.com/5752560/42407435-2a823086-81ef-11e8-9d27-81d77c65c4de.png)